roles/sft-server/molecule/default/verify.yml

```diff
@@ -2,8 +2,65 @@
 # This is an example playbook to execute Ansible tests.
 
 - name: Verify
-  hosts: all
+  hosts: localhost
   tasks:
-  - name: Example assertion
-    assert:
-      that: true
+    - name: include molecule variables
+      include_vars: shared_vars.yml
+
+    # TODO: Make this return 200, 400 is good enough to verify that the request
+    # is forwarded _somewhere_.
+    - name: SFTD is accessible
+      uri:
+        url: "https://{{ sft_fqdn }}"
+        timeout: 10
+        validate_certs: "{{ not certbot_in_test_mode }}"
+      register: response
+      failed_when: response.status != 400
+
+    - name: Temporary file for client private key
+      tempfile:
```

Comment:
Yes, there is nothing to configure there.

Reply:
Would `tempfile: {}` be better?

Reply:
> Would `tempfile: {}` be better?

Yes. I think that would settle any doubts that this might have been by accident. Would you mind elaborating on why you used `tempfile` here (and every other task related to TLS in this PR)?

Reply:
To make sure we don't regenerate certificates, the code tries to download them. If they don't exist, create them and upload them. The create step using `openssl_*` modules always puts them in a file, for this I need a temporary file.